src/evolution.jl

```diff
@@ -73,14 +75,28 @@ function remove_event(chromo::String; idx = nothing)::String
               (r_idx < length(chromo) && chromo[r_idx+1] == '|')
         l_bracket, pipe, r_bracket, plus = find_brackets(chromo, idx)
         r_shift = plus ? 2 : 1
-
-        chromo[1:l_bracket-1] * chromo[pipe+1:r_bracket-1] * chromo[r_bracket+r_shift:end]
+        if pipe == nothing
+            if r_bracket - l_bracket == 3
+                chromo[1:l_bracket-1] * chromo[idx+1] * chromo[idx+3:end]
+            else
+                chromo[1:l_bracket] * chromo[idx+1:end]
+            end
+        else
+            chromo[1:l_bracket-1] * chromo[pipe+1:r_bracket-1] * chromo[r_bracket+r_shift:end]
+        end
     elseif (idx > 1 && chromo[idx-1] == '|') ||
            (r_idx < length(chromo) && chromo[r_idx+1] == ')')
         l_bracket, pipe, r_bracket, plus = find_brackets(chromo, idx)
         r_shift = plus ? 2 : 1
-
-        chromo[1:l_bracket-1] * chromo[l_bracket+1:pipe-1] * chromo[r_bracket+r_shift:end]
+        if pipe == nothing
```

Comment:
Czy skoro po lewej stronie od eventu znajduje się pipe `chromo[idx-1] == '|')` to czy pipe może być nothingiem?